pkg/cmd/cpoperator/operator-config.go

```diff
@@ -92,6 +95,17 @@ func (c *ControlPlaneOperatorConfig) Config() *rest.Config {
 	return c.config
 }
 
+func (c *ControlPlaneOperatorConfig) NamespacedKubeInformers() kubeinformers.SharedInformerFactory {
+	if c.kubeInformers == nil {
+		c.kubeInformers = kubeinformers.NewSharedInformerFactoryWithOptions(c.KubeClient(), controllers.DefaultResync, kubeinformers.WithNamespace(c.namespace))
```

Comment:
this will cause our load to go up again.... Can we leader elect inside the cluster (since kube-apiserver will be operational) and have the informers be inside the cluster as well?

Reply:
That informer actually is not getting used. I will remove it. Thx!

Reply:
Removed the informer.